src/metabase/models/humanization.clj

```diff
@@ -16,75 +19,111 @@
   #{"id" "url" "ip" "uid" "uuid" "guid"})
 
 (defn- capitalize-word [word]
-  (if (contains? acronyms (s/lower-case word))
-    (s/upper-case word)
-    (s/capitalize word)))
-
-(defn- name->human-readable-name:advanced
-  "Implementation of `name->human-readable-name` used if the Setting `enable-advanced-humanization` is `false`."
-  ^String [^String s]
-  (when (seq s)
-    ;; explode string on spaces, underscores, hyphens, and camelCase
-    (s/join " " (for [part  (s/split s #"[-_\s]+|(?<=[a-z])(?=[A-Z])")
-                      :when (not (s/blank? part))
-                      word  (dedupe (flatten (infer-spaces part)))]
-                  (capitalize-word word)))))
-
-(defn- name->human-readable-name:simple
-  "Implementation of `name->human-readable-name` used if the Setting `enable-advanced-humanization` is `true`."
-  ^String [^String s]
-  ;; explode on hypens, underscores, and spaces
-  (when (seq s)
-    (s/join " " (for [part  (s/split s #"[-_\s]+")
-                      :when (not (s/blank? part))]
-                  (capitalize-word part)))))
-
-
-(declare enable-advanced-humanization)
-
-(defn name->human-readable-name
-  "Convert a string NAME of some object like a `Table` or `Field` to one more friendly to humans.
-
-    (name->human-readable-name \"admin_users\") -> \"Admin Users\"
-
-   (The actual implementation of this function depends on the value of `enable-advanced-humanization`; by default,
-   `name->human-readable-name:advanced` is used)."
-  ^String [^String s]
-  ((if (enable-advanced-humanization)
-      name->human-readable-name:advanced
-      name->human-readable-name:simple) s))
+  (if (contains? acronyms (str/lower-case word))
+    (str/upper-case word)
+    (str/capitalize word)))
+
```

Comment:
Extra line

Reply:
I put that in there to break the namespace up a bit and group together the related stuff but if you think that doesn't make sense I can take it out